src/Compilers/Core/AnalyzerDriver/AnalyzerDriverHelper.cs

```diff
@@ -459,212 +434,13 @@ internal static bool CanHaveExecutableCodeBlock(ISymbol symbol)
             }
         }
 
-        private static void VerifyArguments(
```

Comment:
We don't need to verify arguments as AnalyzerDriverHelper.Execute*Actions are internal.